### DISCUSSION TITLE
Replace controller specs with requests specs in solidus_starter_frontend

### DISCUSSION THREAD

**Discussion by peterberkenbosch** [solidusio/solidus#4716] [New Features or Ideas]
Looks like there was some understandable copy/paste from the frontend gem in the starter_frontend project, with the current best practises I think we should start by replacing the controller specs with request specs and remove some specs that are not testing actual code in the receiving project. (For example base_controller_spec.rb)

Curious to hear thoughts..

**Comment by waiting-for-dev** [comment]
💯 that makes total sense

**Comment by kennyadsl** [comment]
Hey @peterberkenbosch!

The controller specs coming from solidus_frontend have been converted to request specs already with https://github.com/solidusio/solidus_starter_frontend/pull/90. More recently, we decided also to incorporate `solidus_auth_devise` code into the new starter frontend, and we missed converting its controller specs into requests. 

Nice catch, it's definitely something that needs to be done. Thanks for reporting!
